.github/release.yml

```
@@ -0,0 +1,17 @@
+changelog:
+  categories:
+    - title: Breaking Changes 🛠
```

```suggestion
    - title: Breaking Changes
```

Not a fan of emojis on changelogs.